scripts/python/airflow/dags/contralory/contralory_page.py

```diff
@@ -0,0 +1,127 @@
+import json
+import sys
+import re
+import os
+
+from typing import List
+from bs4 import BeautifulSoup
+from pathlib import Path
+from fnmatch import fnmatch
+import requests
+import pickle
+
+def find(path: str, pattern: str = '*.pdf') -> List[str]:
+    """
+    busca todos los pdfs del directorio
+    """
+    result = []
+    for _, _, files in os.walk(path):
+        for name in files:
+            if fnmatch(name, pattern):
+                result.append(os.path.join(name))
+    return(result)
+
+def contraloria_get_urls(contraloria_url: str, ti, **kwargs) -> List[str]:
+    '''
+    Dada la pagina web de la contraloria publica del paraguay:
+    Obtener lista de URLs que apunta a los PDFs
+    '''
+    s = requests.Session()
+    urlist = []
+    
+    print(f"Consiguiendo links de: {contraloria_url}")
+    
+    r = s.get(contraloria_url)
+    
+    if(r.ok):
+        page_content = r.content
+        page_soup = BeautifulSoup(page_content, 'html.parser')
+        imput_list = page_soup.findAll("input", {'type': 'hidden', 'value': '1'})
+        magic_number = imput_list[0].get('name')
+        post_data = {magic_number: '1', 'limit': '0'}
+        r_url = s.post(contraloria_url, data=post_data)
+
+        if(r_url.ok):
+            URLs = BeautifulSoup(r_url.content, 'html.parser')
+            for btn in (URLs.findAll('a', {'class': 'btn btn-success'})):
+                urlist.append(contraloria_url + btn.get('href')[1:])
+            print(f'Obtenidos: {str(len(urlist))} Links')
+            ti.xcom_push(key='some_failure', value=False)
+            return(urlist)
+        else:
+            ti.xcom_push(key='some_failure', value=True)
+            r_url.raise_for_status()
+    else:
+        ti.xcom_push(key='some_failure', value=True)
```

Comment:
This is not very useful to debug the issue, add the response status code and all the headers